core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -880,18 +1036,303 @@ export class LoggedInUser extends BaseUser {
+    );
+
+    await Promise.all([
+      this.page.waitForNavigation(),
+      await this.clickOn('Create an Exploration'),
+    ]);
+
+    expect(this.page.url()).toBe(
+      'http://localhost:8181/creator-dashboard?mode=create'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "Browse our Expectations" on the Creator Guidelines page.
+   */
+  async clickBrowseOurExpectationsLinkOnCreatorGuidelinesPage(): Promise<void> {
+    await this.page.waitForXPath(
+      '//a[contains(text(),"Browse our Explorations")]'
+    );
+
+    await Promise.all([
+      this.page.waitForNavigation(),
+      await this.clickOn('Browse our Explorations'),
+    ]);
+
+    expect(this.page.url()).toBe('http://localhost:8181/community-library');
+  }
+
+  /**
+   * Clicks the link with the text "Browse out Lessons" on the Teach page.
+   */
+  async clickBrowseOurLessonsOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'Browse our Lessons',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "Access the Android app" on the Teach page.
+   */
+  async clickAccessTheAndroidAppOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'Access the Android app',
+      'Creator Guidelines in the About Menu on navbar',
+      androidUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "VISIT CLASSROOM" on the Teach page.
+   */
+  async clickVisitClassroomOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'VISIT CLASSROOM',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "BROWSE LIBRARY" on the Teach page.
+   */
+  async clickBrowseLibraryOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'BROWSE LIBRARY',
+      'Creator Guidelines in the About Menu on navbar',
+      communityLibraryUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "Explore Lessons" on the Teach page.
+   */
+  async clickExploreLessonsOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'Explore lessons',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link on the Terms page about the CC-BY-SA 4.0 license.
+   */
+  async clickLinkToLicenseOnTermsPage(): Promise<void> {
+    await this.page.waitForXPath('(//a[contains(text(),"here")])[1]');
+    const [link] = await this.page.$x('(//a[contains(text(),"here")])[1]');
+    await Promise.all([this.page.waitForNavigation(), await link.click()]);
+
+    expect(this.page.url()).toBe(
+      'https://creativecommons.org/licenses/by-sa/4.0/legalcode'
```

Comment:
Again, why don't store it as a constant?

Reply:
Done